docs/source/managed-federation/setup.mdx

```diff
@@ -71,33 +69,37 @@ const gateway = new ApolloGateway({
 
 This option is specific to _non_-managed federation, in which supergraph schema composition is performed via the Rover CLI.
 
-With managed federation, composition is instead performed by _GraphOS_, and the gateway regularly polls for an updated schema. This enables you to add, remove, and modify your subgraphs _without_ needing to restart your gateway.
+With managed federation, composition can instead be performed by _GraphOS_, where the gateway regularly polls for an updated schema. This enables you to add, remove, and modify your subgraphs _without_ needing to restart your gateway.
```

Comment:
Non-blocking, just curious about the change from "is" to "can" across this page. With managed federation isn't the point that composition _is_ instead handled by GraphOS?

Reply:
The distinction is that "managed federation" in general manages one or all aspects of CI/CD for federation, and composition may or may not be managed (`can` vs. `is).

That said, you're right that this sentence is focused on what GraphOS provides. Rewriting to `With federation managed by GraphOS, composition is performed and ...`